### PR TITLE
Fixed leftover youtube-dl references.

### PR DESCRIPTION
**Pull request by anthonyrocom**

### CODE REVIEW

File: catt/cli.py [hunk 1]
```diff
@@ -180,10 +180,11 @@ def cli(ctx, device):
 @click.option(
     "-y",
     "--ytdl-option",
+    "--ytdlp-option",
```
Comment (skorokithakis) [inline catt/cli.py:183]: I don't think this needs to be changed, and it would break a bunch of stuff. It can stay as it was, I think.

Comment (anthonyrocom) [reply catt/cli.py:183]: Agreed

Comment (skorokithakis) [reply catt/cli.py:183]: Can you change it back though?

Comment (skorokithakis) [reply catt/cli.py:183]:
```suggestion
    "--ytdl-option",
```

**Comment by skorokithakis** [comment]
@anthonyrocom you've tested this and it works properly, right?

**Comment by anthonyrocom** [comment]
> @anthonyrocom you've tested this and it works properly, right?

Is YouTube working? Yes it's working and I tested that.

**Review by skorokithakis — APPROVED** [review]

File: catt/cli.py [hunk 2]
```diff
@@ -180,10 +180,11 @@ def cli(ctx, device):
 @click.option(
     "-y",
     "--ytdl-option",
+    "--ytdl-option",
```
Comment (skorokithakis) [inline catt/cli.py:183]:
```suggestion
```

**Review by skorokithakis — APPROVED** [review]

**Pull request merged**